OmniBLE/OmnipodCommon/MessageBlocks/VersionResponse.swift

```diff
@@ -13,7 +13,7 @@ fileprivate let assignAddressVersionLength: UInt8 = 0x15
 fileprivate let setupPodVersionLength: UInt8 = 0x1B
 
 let erosProductId: UInt8 = 2        // 2nd gen Eros Omnipod (PM=PI=2.x.y)
-let dashProductId: UInt8 = 4        // 4th gen Dash Omnipod (firmware version 4.x.y)
+let dashProductId: UInt8 = 4        // Dash NXP BLE (firmware 3.x.y) or TWI BOARD (firmware 4.x.y)
```

Comment:
Could you remove these, change them into comments, or move them to be local to tests?

Reply:
Oh, I see these *are* being used in the code. These should be a type instead of wide scope constants, but I'll take this PR for now, and that can be addressed later.